de.sebastianruziczka.looksyk.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
post corrections this file should be taken upstream.

Reply:
I guess, I have fixed all comments.